plugins/community/mypy-protobuf-grpc/v3.4.0/requirements.txt

```diff
@@ -0,0 +1,3 @@
+mypy-protobuf==3.4.0
+protobuf==4.21.9
+types-protobuf==4.21.0.1
```

Comment:
Created this with `pip freeze` in the container after installing. This should ensure consistent builds.